pkg/api/convertertoapi.go

```diff
@@ -635,6 +635,10 @@ func convertAddonsToAPI(v *vlabs.KubernetesConfig, a *KubernetesConfig) {
 				MemoryLimits:   v.Addons[i].Containers[j].MemoryLimits,
 			})
 		}
+
```

Comment:
@sozercan I added the below lines to adapt the KubernetesAddons default assignment convenience function to include `Config` key/vals as well